cmake/CMakeDetermineRustCompiler.cmake

```diff
@@ -0,0 +1,25 @@
+
+if(NOT CMAKE_Rust_COMPILER)
```

Comment:
Ideally I think it'd be nice to turn on cdb compiling if Rust is detected on the host, otherwise the target is skipped. Similar how test target is treated depending on whether check can be found or not.